x-pack/packetbeat/npcap/installer/LICENSE

```diff
@@ -0,0 +1,229 @@
+--------------------------------------------------------------------------------
```

Comment:
This is a place-holder file. It should probably also have a link to the canonical license file to avoid risk of errors.